.github/workflows/release.yml

```diff
@@ -116,12 +125,19 @@ jobs:
             chmod a+x "${archive}/phylum"
             zip -r "${archive}.zip" "${archive}"
           done
+          for archive in phylum-*-windows-*/;
+          do
+            archive=$(echo "${archive}" | sed -e "s/\/$//")
+            mv "${archive}/phylum.exe" "${archive}.exe"
+          done
 
       - name: Upload release artifacts
         uses: actions/upload-artifact@834a144ee995460fba8ed112a2fc961b36a5ec5a # v4.3.6
         with:
           name: release-archives
-          path: phylum-*.zip
+          path: |
+            phylum-*.zip
+            phylum-*.exe
```

Comment:
Because of this new name/extension, the Windows artifact won't be included in the release assets. The `Upload release assets` step of the `Release` job will need to be updated if that is the goal.